scripts/docker/buildpack-deps/Dockerfile.ubuntu2404.arm

```
@@ -104,6 +104,7 @@ RUN <<-EOF
 EOF
 
 FROM base
+COPY --from=libraries /usr/local/lib/libevmone* /usr/lib
```

```suggestion
COPY --from=libraries /usr/local/lib /usr/local/lib
```

This should be fine, /usr/local/lib is typically in the PATH.

---

True. I wanted to make sure at first and then check if this was fine also and ended up forgetting.